airbyte-integrations/connectors/source-first-connector/bin/.gitignore

```diff
@@ -0,0 +1 @@
+/com/
```

Comment:
not sure how this works, because it has absolute path, where did you get this line?